### PR TITLE
Document local dev setup

### PR DESCRIPTION
**Pull request by anothertobi**
## Summary

* Document local dev setup

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,41 @@ make autorest
...
+- Stardog image registry credentials
+- Stardog license
+
+### Init local dev environment
+
+```
+minikube start
+
+# Create values.yaml (fill in actual image registry credentials)
+<<EOF > values.yaml
```
Comment (simu) [inline README.md:43]: Are you missing a `cat` here? The provided syntax creates an empty `values.yaml` for me (Linux/bash)

File: README.md [hunk 2]
```diff
@@ -23,3 +23,41 @@ make autorest
...
+
+## Local Development
+
+### Prerequisites
+
+- minikube
+- helm
+- kubectl
+- Stardog image registry credentials
+- Stardog license
```
Comment (simu) [inline README.md:35]: Do we have any documentation (internal is fine) regarding where to find this information?

Comment (anothertobi) [reply README.md:35]: Added a link to an internal wiki page

File: README.md [hunk 3]
```diff
@@ -23,3 +23,43 @@ make autorest
...
+
+See https://wiki.vshn.net/x/PIuVEw (VSHN internal) on how to retrieve registry credentials and licenses for Stardog.
```
Comment (simu) [inline README.md:37]:
```suggestion
See [the VSHN wiki (internal)](https://wiki.vshn.net/x/PIuVEw) for details on how to retrieve registry credentials and licenses for Stardog.
```

**Review by simu — APPROVED** [review]
LGTM now. Disclaimer: I didn't fully test the instructions.

**Pull request merged**